components/dex.libsonnet

```diff
@@ -70,6 +76,24 @@
               name: 'storage',
               persistentVolumeClaim: { claimName: dex.config.name },
             },
+            {
+              name: 'tls',
+              secret: {
+                // this secret is created automatically by OpenShift when
+                // the service has the appropriate annotation
+                secretName: dex.config.tlsSecret,
+                items: [
+                  {
+                    key: 'tls.crt',
```

Comment:
we typically like to make the keys configurable in jsonnet (e.g. [0]) because not every certificate sercret creator uses the same key names. 

[0] https://github.com/observatorium/observatorium/blob/master/jsonnet/lib/observatorium-api.libsonnet#L204-L216

Reply:
Done